cc3d/player5/Plugins/ViewManagerPlugins/SimpleTabView.py

```diff
@@ -221,7 +221,8 @@ def setup_logging(self):
         """
         log_level_val = getattr(CompuCell, Configuration.getSetting("LogLevel"))
         if log_level_val != self.log_level:
-            CompuCell.Logger.enableConsoleLogging(log_level_val)
+            logger = CompuCell.CC3DLogger.get()
+            logger.enableConsoleLogging(log_level_val)
```

Comment:
Is there a mechanism to disable console logging?

Reply:
console logging is always on you just choose the level you want. I do not want to give too many options to users because then things get confusing. Essentially by selecting the log-level they can control how much information they get and there are few levels that do not output any information unless something goes really wrong. This behavior makes sense to me